Dockerfile

```
@@ -20,15 +26,22 @@ RUN npm ci --no-audit
 # Set commit hash
 RUN if [[ $IS_STABLE == "0" ]] ; then NUXT_ENV_COMMIT=$(git rev-parse HEAD) ; fi
 
+
 # Build client
 RUN npm run build
 
+
 # Deploy built distribution to nginx
 FROM nginx:alpine
+
 COPY --from=build /app/src/dist/ /usr/share/nginx/html/
+
+COPY --from=build /app/.docker/nginx.conf /etc/nginx/conf.d/default.conf 
+
 EXPOSE 80
 
 # Set labels
 LABEL maintainer="Jellyfin Packaging Team - packaging@jellyfin.org"
 LABEL org.opencontainers.image.source="https://github.com/jellyfin/jellyfin-vue"
 LABEL org.opencontainers.image.description "Commit: ${NUXT_ENV_COMMIT}"
+LABEL org.opencontainers.image.description "History Router Mode: ${HISTORY_ROUTER_MODE}"
```

Doing duplicates remove the previous one: https://docs.docker.com/config/labels-custom-metadata/#label-keys-and-values

```suggestion
LABEL org.opencontainers.image.description="Commit: ${NUXT_ENV_COMMIT} History router mode: ${HISTORY_ROUTER_MODE}"
```